discovery-provider/src/queries/get_notifications.py

```diff
@@ -57,7 +49,7 @@
 LEFT JOIN user_seen on
   user_seen.seen_at >= n.timestamp and user_seen.prev_seen_at < n.timestamp
 WHERE
-  ((ARRAY[:user_id] && n.user_ids) OR (n.type = 'announcement' AND n.timestamp > (SELECT created_at FROM user_created_at))) AND
```

Comment:
we need this filter still don't we? so you dont see old ancments

Reply:
my b i committed based on a release branch diff